aiohttp/test_utils.py

```diff
@@ -78,6 +78,8 @@ def _make_factory(self):
         pass
 
     def make_url(self, path):
+        if isinstance(path, URL):
```

Comment:
I thinks yes, it's just testing tool.
`.startswith()` tests for relative URL.
The same could be done by `assert not URL(path).is_absolute()`.
Not sure if we need this check but it is harmless.